### PR TITLE
mirrors.yml: update the mirrors url

### PR DESCRIPTION
**Pull request by Plectranthus-fg**
rm CQU OSS mirror: unavailable
cqupt mirror: stop updating
geekpie mirror: update mirror address, "https://mirrors.cqu.edu.cn/aosc/" -> "https://mirrors.geekpie.club/anthon/"
ustc mirror: add ipv4 and ipv6 url
tuna mirror: add ipv4 and ipv6 url

### CODE REVIEW

File: mirrors.yml [hunk 1]
```diff
@@ -1,15 +1,12 @@
 origin:
     desc: AOSC main repository (hosted by BlueT, and National Chi Nan University)
```
Comment (Yexiaoxing) [inline mirrors.yml:2]: no longer bluet

File: mirrors.yml [hunk 2]
```diff
@@ -25,6 +22,18 @@ tencent:
 tuna:
     desc: Tsinghua University TUNA Association OSS Mirror
     url: https://mirrors.tuna.tsinghua.edu.cn/anthon/
+tuna4:
+    desc: Tsinghua University TUNA Association OSS Mirror
```
Comment (Yexiaoxing) [inline mirrors.yml:26]: add some desc like `(IPv4 only)`?

Comment (LionNatsu) [inline mirrors.yml:26]: There should be different descriptions for the 4 and the 6. For example, `(IPv4 only) (IPv6 only)`.

File: mirrors.yml (hunk 1 above)
Comment (LionNatsu) [reply mirrors.yml:2]: Is it still hosted by National Chi Nan University? If no, what is it?

Comment (Yexiaoxing) [reply mirrors.yml:2]: `(sponsored by xTom.com)`

Comment (Plectranthus-fg) [reply mirrors.yml]: only sponsored by xTom.com ?

**Pull request merged**